content/python/concepts/math-module/terms/math-acos/math-acos.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'math.acos()'
+Description: 'Returns the inverse cosine (in radians) of a value between -1 and 1.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`math.acos()`** function returns the inverse cosine (in radians) of a value between -1 and 1.
+
+## Syntax
+
+```pseudo
+math.acos(n)
+```
+

```

Comment:
```suggestion

- `x`: A number between `-1` and `1` representing the cosine value for which the arc cosine is to be calculated.

```

Reply:
The parameter needs to be described, feel free to change the description this is just for reference